src/modules/directory.rs

```diff
@@ -321,6 +325,10 @@ fn to_fish_style(pwd_dir_length: usize, dir_string: String, truncated_dir_string
         .join("/")
 }
 
+fn from_slash(path: &str) -> String {
```

Comment:
Similar complaint here: I don't know what `from_slash` is supposed to do from its name. If we can't get a more descriptive name that isn't a mouthful (because the only names I can think of are way too long, like "convert_to_platform_path_separators"), I'd like to see a doc comment that describes what it does.